index.js

```diff
@@ -76,6 +76,14 @@ module.exports = function Expirer(timeoutMs, db, checkIntervalMs) {
 
 	expirer.touch = expirer.emit.bind(expirer, 'touch')
 	expirer.forget = expirer.emit.bind(expirer, 'forget')
+	expirer.createIfNotExists = function createIfNotExists(key) {
+		db.get(key, function (err, value) {
+			if (err && err.notFound) {
+				console.log('creating ' + key)
+				expirer.emit('touch', key)
```

Comment:
There's a race condition here where some other code could call `touch` in between the `get` on line 80 and here, but I think that's acceptable.  It might be worth adding a comment to note it (and note that we're okay with it) though.